sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -359,7 +359,7 @@ def compare_split_points(file_name):
       source_test_utils.assert_split_at_fraction_exhaustive(splits[0].source)
 
     if self.use_fastavro:
-      file_name = self._write_data(count=200, sync_interval=2)
+      file_name = self._write_data(count=5, sync_interval=2)
```

Comment:
Could you add sync_interval as an optional (defaulting to None) argument to _write_data and remove the branching here?

Reply:
Done

Reply:
We default sync_interval to respective defaults in both Avro/FastAvro implementations.